openupgradelib/openupgrade.py

```diff
@@ -825,6 +825,17 @@ def rename_models(cr, model_spec):
             "UPDATE ir_filters SET model_id = %s "
             "WHERE model_id = %s", (new, old,),
         )
+        logged_query(
+            cr, """
+            UPDATE ir_property
+            SET res_id = regexp_replace(
+                res_id, %(old_pattern)s, %(new_pattern)s
+            )
+            WHERE res_id ~ %(old_pattern)s""", {
+                'old_pattern': r"^%s,[ ]*([0-9]*)" % old,
```

Comment:
Why the `[ ]*`? It shouldn't be any space in between according ORM

Anyways, I think you don't need regular expressions, as the `WHERE... like` avoids the possibility of matching a substring instead the beginning of the field:
```python
logged_query(
    env.cr,
    """UPDATE ir_property
    SET res_id = replace(res_id, %(old_string)s, %(new_string)s)
    WHERE res_id like %(old_pattern)s""",
    {
        "old_pattern": "%s,%%" % old,
        "old_string": "%s," % old,
        "new_string": "%s," % new,
    },
)
```

Reply:
Well, I think we can improve both :wink: (in separate commits). I suppose using regexp will be always slower.

Reply:
Done!